texture_manager.h

```diff
@@ -55,16 +55,16 @@ void texture_manager_destroy(texture_manager *manager);
 void texture_manager_clear_textures(texture_manager *manager, bool clear_all);
 void texture_manager_free_texture(texture_manager *manager, texture_2d *tex);
 void texture_manager_touch_texture(texture_manager *manager, const char *url);
-void texture_manager_set_use_halfsized_textures();
```

Comment:
Similarly here, we should take a `texture_manager*` and have a `use_halfsized` flag on the texture_manager itself.